### DISCUSSION TITLE
Please exclude Nightscout password and user ID from the status page

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#2291] [Ideas]
Posts like this, revealing password and user ID, are normal on facebook.
![Screenshot 2022-08-23 212646](https://user-images.githubusercontent.com/51497406/186297453-1d3b31e2-a1cd-4cf8-a214-1738eec5b5e3.png)

I have covered the user ID and password in my screenshot.  But, the person who posted this message had left everything readable. 
 Users post asking for help not realizing that they are revealing their id and password.

Can we change this so that everything remains except only the user ID and password?
So, looking at the message, one would still know that there is an issue with upload host resolution, in this case for example.
And the user can always see what base url they have entered going to the REST-API page.  

The result will be that less passwords and user IDs will be unintentionally revlealed.

**Comment by Lupum1001** [comment]
It's just the URL to their (unauthenticated) Nightscout Instance. In order to authenticate requests, there's an additional part of the URL that is not shown in that error message (the API "secret").
The full URL to actually make API calls would be:
https://**1234myApiSecretCode1234**@myAppName.azurewebsites.net/api/v1/

Now, that being said, "myAppName" _could_ be considered sensitive information, since it lets you view that person's website, even if it's read-only. It's arguably analogous to a username, but only for the one app, rather than their entire Azure account.

**Comment by Lupum1001** [comment]
Do you know of any other sensitive information worth hiding on the status page? I don't use all the plugins, and it's been a few months since I looked at all the potential MegaStatus page options.

I mean, the Dexcom Transmitter ID is sensitive information, though less useful to the wider internet. The Bluetooth device Advertises the last 2 characters at the end of its name, so maybe hide the first 4 characters by default with some kind of toggle in place?
